src/patternfly/components/Toolbar/toolbar-overflow-menu-example.hbs

```diff
@@ -1,5 +1,5 @@
-{{#> toolbar-item toolbar-item--modifier="pf-m-overflow-menu"}}
-  {{#> overflow-menu overflow-menu-button--aria-label="Expand overflow menu" overflow-menu--id=(concat toolbar--id '-overflow-menu') dropdown--IsExpanded=toolbar-overflow-menu-example--IsExpanded overflow-menu--modifier=toolbar-demo-overflow-menu--modifier overflow-menu--attribute=toolbar-demo-overflow-menu--attribute}}
+{{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-overflow-menu ' toolbar-overflow-menu-example--modifier)}}
```

Comment:
for something like this you may call this something specific to the item, so that you can add this to other elements in the same hbs file. So something like `toolbar-overflow-menu-example-toolbar-item--modifier`